models/binary_neuron.h

```diff
@@ -273,7 +273,7 @@ template < class TGainfunction >
 inline SignalType
 binary_neuron< TGainfunction >::receives_signal() const
 {
-  return BINARY;
+  return BINARY | STEPWISE_CONSTANT;
```

Comment:
@tobikausk This does not compile because the bitwise or produces an integer that cannot be converted into an enumeration value of type `SignalType`.  The corresponding operation works in the definition of the enumeration, because there an integer is expected on the right hand side of `=`.

@jougs What would be the best way to fix this? `bitset`?